core/src/container.cpp

```diff
@@ -363,7 +363,8 @@ void SerialContainer::onNewSolution(const SolutionBase& current) {
 	solution.reserve(children.size());
 	for (auto& in : incoming.solutions) {
 		for (auto& out : outgoing.solutions) {
-			InterfaceState::Priority prio(in.first.size() + 1 + out.first.size(), in.second + current.cost() + out.second);
+			InterfaceState::Priority prio(static_cast<unsigned int>(in.first.size() + 1 + out.first.size()),
```

Comment:
Why is clang complaining here? `size()` yields an unsigned type. Is it picky about the (signed?) `1`?
What about:
```suggestion
			InterfaceState::Priority prio(in.first.size() + 1u + out.first.size(),
```

Reply:
It's actually about `Priority` expecting an `unsigned int` but this expression computing an `unsigned long`.
The alternative would be to adjust the `Priority` class, but I don't think that's necessary.